### ISSUE TITLE
Run the code in unix environment

### ISSUE THREAD

**Issue by zjjdcrn** [buttomnutstoast/MooneyFaceGenerator#1]
Hi,
A little question. I wonder whether the generator scripts could work in the unix environment. 
Thank you very much

**Comment by buttomnutstoast** [comment]
Yes, I think so, the code is runned on RedHat6. It should be also runnable on other Linux systems.
